test/zone_test.rb

```diff
@@ -47,14 +47,6 @@ def test_zone_has_config_with_provider
     assert_instance_of Zone::Config, zone.config
   end
 
-  def test_zone_is_invalid_if_provider_does_not_support_record_type
-    zone = Zone.find('empty.com')
-    assert_equal 'DynECT', zone.config.provider
-    zone.records = [Record::ALIAS.new(fqdn: zone.name, ttl: 60, cname: "alias.#{zone.name}")]
-
-    refute_predicate zone, :valid?
-  end
```

Comment:
I'd like to keep the test of this logic somehow, but we can't use ALIAS anymore for testing 😕